cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -113,6 +114,7 @@ func main() {
 	flag.StringVar(&flagResponseTypeSuffix, "response-type-suffix", "", "The suffix used for responses types.")
 	flag.BoolVar(&flagAliasTypes, "alias-types", false, "Alias type declarations of possible.")
 	flag.BoolVar(&flagInitialismOverrides, "initialism-overrides", false, "Use initialism overrides.")
+	flag.StringVar(&flagAdditionalInitialisms, "addition-initialisms", "", "Comma separated list of additional initialisms to use.")
```

Comment:
This maybe

```suggestion
	flag.StringVar(&flagAdditionalInitialisms, "additional-initialisms", "", "Comma separated list of additional initialisms to use.")
```

Or


```suggestion
	flag.StringVar(&flagAdditionalInitialisms, "extra-initialisms", "", "Comma separated list of additional initialisms to use.")
```


Addition sounds strange

Reply:
misspelled it, sorry! fixed :)